hstream-http-server/src/HStream/HTTP/Server/Connector.hs

```diff
@@ -84,10 +84,22 @@ hstreamConnectorToConnectorBO (HSP.Connector connectorId (HSP.Info sqlStatement
 hstreamConnectorNameIs :: T.Text -> HSP.Connector -> Bool
 hstreamConnectorNameIs name (HSP.Connector connectorId _ _) = (cbytesToText connectorId) == name
 
--- removeConnectorHandler :: HS.LDClient -> String -> Handler Bool
--- removeConnectorHandler ldClient name = do
---   liftIO $ removeConnector ldClient (mkConnectorName $ ZDC.pack name)
---   return True
+removeConnectorHandler :: HS.LDClient -> Maybe ZK.ZHandle -> String -> Handler Bool
+removeConnectorHandler ldClient zkHandle name = do
+  res <- liftIO $ do
+    connectors <- HSP.withMaybeZHandle zkHandle HSP.getConnectors
+    case find (hstreamConnectorNameIs (T.pack name)) connectors of
+      Just (HSP.Connector connectorId _ (HSP.Status status _)) -> do
+        if HSP.Terminated == status
+          then
+            do
+              HSP.withMaybeZHandle zkHandle $ HSP.removeConnector connectorId
+              return True
+          else
+            -- TODO: should return detailed reason
+            return False
```

Comment:
If the connector has not been terminated, a `STILLRUNING` exception will be thrown when `removeConnector`, same with `removeQuery`

Reply:
`HSP.withMaybeZHandle zkHandle $ HSP.removeQuery (T.pack connector)`